### DISCUSSION TITLE
Who was lead singer of Led Zeppelin?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#686] [Galaxy Trivia] [answered]
**Bonus:** What was the original name of the band?

**Comment by DevElkami** [comment, marked as answer]
Robert Plant. Original name of the group is the New Yardbirds.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [🥉 Galaxy Brain](https://github.com/DevElkami?achievement=galaxy-brain&tab=achievements) badge. Keep up the great work and unlock the Silver 🥈 badge next by answering 16 questions. Don't forget to post your own questions too! 😊🎉

**Comment by peterbaccaro** [comment]
**Robert Plant** was the lead singer of Led Zeppelin.
The original name of the group was the **New Yardbirds**.
The name Led Zeppelin came from a comment made by Keith Moon of The Who, who once said the band would go down like a lead balloon.

**Comment by thuongtruong1009** [comment]
The lead singer of Led Zeppelin was Robert Plant.

The original name of the band was "The New Yardbirds". After the original Yardbirds disbanded in 1968, guitarist Jimmy Page recruited Plant, along with John Paul Jones (bass, keyboards) and John Bonham (drums), to form a new band. The band was initially named The New Yardbirds, but after a short time, they changed their name to Led Zeppelin, and the rest is history.
